docs/flutter/v2/features/mute.mdx

```diff
@@ -9,8 +9,8 @@ Mute is something that applies to both audio and video. When you mute audio, you
 
 This example shows implementation of three methods namely : 
 
-### 1. switchAudio : To switch local peer's audio on/off.
-### 2. switchVideo : To switch local peer's video on/off.
+### 1. toggleMicMuteState : To switch local peer's audio on/off.
```

Comment:
- add code snippet showing usage
- add comments showing the mute states before & after calling the APIs
- add code snippet + comments on how to get the current mute state